src/decorators.ts

```diff
@@ -23,36 +26,10 @@ export interface DBOSLifecycleCallback {
   logRegisteredEndpoints?(): void;
 }
 
-const lifecycleListeners: DBOSLifecycleCallback[] = [];
-export function registerLifecycleCallback(lcl: DBOSLifecycleCallback) {
-  if (!lifecycleListeners.includes(lcl)) lifecycleListeners.push(lcl);
-}
-export function getLifecycleListeners() {
-  return lifecycleListeners as readonly DBOSLifecycleCallback[];
-}
-
 // Middleware installation
 export interface DBOSMethodMiddlewareInstaller {
   installMiddleware(methodReg: MethodRegistrationBase): void;
 }
-let installedMiddleware = false;
-const middlewareInstallers: DBOSMethodMiddlewareInstaller[] = [];
-export function registerMiddlewareInstaller(i: DBOSMethodMiddlewareInstaller) {
-  if (installedMiddleware) throw new TypeError('Attempt to provide method middleware after insertion was performed');
-  if (!middlewareInstallers.includes(i)) middlewareInstallers.push(i);
-}
-export function insertAllMiddleware() {
-  if (installedMiddleware) return;
-  installedMiddleware = true;
-
-  for (const [_cn, c] of classesByName) {
-    for (const [_fn, f] of c.reg.registeredOperations) {
-      for (const i of middlewareInstallers) {
-        i.installMiddleware(f);
-      }
-    }
-  }
-}
 
```

Comment:
nit: If we're going to have the `Interfaces` comment, shouldn't we have something here two as the following items are decidedly not interfaces or types